app/src/main/kotlin/com/intive/picover/images/repository/ImagesRepository.kt

```diff
@@ -0,0 +1,32 @@
+package com.intive.picover.images.repository
+
+import android.net.Uri
+import com.google.firebase.storage.FirebaseStorage
+import com.google.firebase.storage.StorageReference
+import javax.inject.Inject
+import kotlin.coroutines.resumeWithException
+import kotlinx.coroutines.suspendCancellableCoroutine
+
+class ImagesRepository @Inject constructor() {
+	private val storageRef = FirebaseStorage.getInstance().reference.child("image")
```

Comment:
to make testing easier it would be better to inject firebase instance

Reply:
Please check new solution. Thank you.

Reply:
please use our DI and inject firebase instance directly